src/platform/user/profile/vap-svc/components/ProfileInformationFieldController.jsx

```diff
@@ -157,8 +155,8 @@ class ProfileInformationFieldController extends React.Component {
     } else if (
       forceEditView &&
       typeof successCallback === 'function' &&
-      prevProps.transactionRequest &&
-      !this.props.transactionRequest
+      prevProps.transactionRequest?.isPending &&
+      !this.props.transactionRequest?.isPending
```

Comment:
@vsa-authd-exp-frontend - I wasn't able to test this change locally, other than in Cypress tests. This `successCallback` is used by our Notice of Disagreement form (not yet in production) and Higher-Level Review form (in production, but behind a feature flag). After merging, I'll manually test this to ensure it's working as expected.

In the Cypress test, I couldn't get the intermediate address validation page to work as expected, when it should have been showing recommendations, it instead called this `successCallback`. But maybe it's because I didn't dig into all the address validation logic.

Reply:
I'm not sure about the Cypress test either since I haven't dug into the address validation logic since joining this team, but it looks like something that is worth investigating to see if we can get better test coverage for that functionality.

I went ahead and created a ticket to look into this. https://github.com/department-of-veterans-affairs/va.gov-team/issues/37077